.golangci.yml

```diff
@@ -72,6 +72,9 @@ linters-settings:
   nolintlint:
     require-explanation: true
     require-specific: true
+  gosec:
+    excludes:
+      - G115 # G115: integer overflow conversion; TODO: currently too many (false) positives in generated code that need to be verified
```

Comment:
I added a global ignore for this one for now; I think most of these are likely false positives in generated code, but would have to be reviewed by someone with more understanding of those parts of the code.